### ISSUE TITLE
Estimation request | Make GEO locations configurable in CMS

### ISSUE THREAD

**Issue by maartenfv** [teamforus/general#927]
The municipality of Ede requestes to remove the map on the provider page. 1. They only want to show the list view. Example Nijmegen:

![image](https://user-images.githubusercontent.com/30211022/185429702-d0e0ae3e-656b-4dab-bd1a-f8fb0afa0068.png)

2. Also there is Geo info on the page of the provider.

![image](https://user-images.githubusercontent.com/30211022/185430202-b3d19937-f1db-499e-90a0-0734416ea161.png)

3. Make the visibility of the image map on the homepage configurable (same as products)

"We zagen afgelopen vrijdag een GEO-kaartje. Dat is niet nodig, deze kan wat ons betreft eruit." Projectleider EDE by mail.

@GerbenBosschieter Question: How much time would it take to make these two points configurable in the CMS. So that Ede can decide themselves if they want to turn it on or of for each of these two points?

**Comment by lexlog** [comment]
**refinement note**

two options
editable:
- CMS 6h ( in dropdown near )

not editable:
- do this with the internal flag ( in qdt-env.js ). est 4h (for @dev-rminds + @aghimpu)
- value in fund configs



out of scope thought - we should think about what to put in CMS, if something is going to be changed only once during lifecycle of webshop. in future CMS can become too complicated, so we should put some not frequent configurations in internal configurations like qdt-env for frontend of database in backend
